buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -813,6 +814,7 @@ class BeamModulePlugin implements Plugin<Project> {
         testcontainers_mysql                        : "org.testcontainers:mysql:$testcontainers_version",
         testcontainers_postgresql                   : "org.testcontainers:postgresql:$testcontainers_version",
         testcontainers_rabbitmq                     : "org.testcontainers:rabbitmq:$testcontainers_version",
+        truth                                       : "com.google.truth:truth:1.0.1",
```

Comment:
I know that Google prefers truth internally. Beam already pulls in hamcrest and JUnit for everything. It may create an annoying situation to have all these very similar frameworks in the project. Is there a special reason to use truth   now?

Reply:
I'm not sure if there is a reason we use truth instead of hamcrest. However, all of our matchers are built on top of truth, so changing this now will be a large refactor. We can consider moving to hamcrest after the migration?

cc @bvolpato